.github/workflows/tag-and-release.yaml

```diff
@@ -50,5 +71,5 @@ jobs:
 
       - name: Publish the capability
         run: |
-          uds zarf package create --confirm -a arm64 -o oci://ghcr.io/defenseunicorns/packages
-          uds zarf package create --confirm -a amd64 -o oci://ghcr.io/defenseunicorns/packages
+          uds zarf package create --confirm -a ${{ matrix.architecture }} -o oci://ghcr.io/defenseunicorns/packages
+          uds zarf package create --confirm -a ${{ matrix.architecture }} --flavor airgap -o oci://ghcr.io/defenseunicorns/packages
```

Comment:
Think this is OK for now, but just to call it out - this is recreating the packages here. What that effectively means is that the package we tested (in `run-airgap-build-test`) is not guaranteed to be the same content as what we publish. Ideally we would:
1. Create package
2. Test package
3. Publish package (using the already created and tested package)